files/build_templates/sonic_debian_extension.j2

```diff
@@ -56,9 +56,7 @@ FILESYSTEM_ROOT_ETC_SONIC="$FILESYSTEM_ROOT_ETC/sonic"
 GENERATED_SERVICE_FILE="$FILESYSTEM_ROOT/etc/sonic/generated_services.conf"
 
 clean_sys() {
-    sudo chroot $FILESYSTEM_ROOT umount /sys/fs/cgroup/*            \
-                                        /sys/fs/cgroup              \
-                                        /sys || true
+    sudo chroot $FILESYSTEM_ROOT umount -R /sys || true
```

Comment:
>sudo chroot $FILESYSTEM_ROOT umount -R /sys || true [](http://example.com/codeflow?start=4&length=51)

Why you need to change this part? Is it an optimization?

Reply:
`/sys/fs/cgroup/*` and `/sys/fs/cgroup` didn't appear to be mount points, and so `umount` would error out. However, just in case they  did get mounted through some code path, I added the `-R` flag to `umount`, so that they would still get umounted if they were indeed mounted.

Reply:
I'm going to revert this part of the change, looks like `/sys/fs/cgroup` does get mounted later in the script.